### DISCUSSION TITLE
Changing the homing sequence

### DISCUSSION THREAD

**Discussion by Switch-Witch** [grblHAL/core#564] [Q&A]
The wiring of my machine I converted to grblhal combined all 3 inductive limit switches to one. 
On mach3 boards this is not that unusual a the homing sequence is Z-Y-X. 
On grblhal it seems that the homing sequence is Z-XY.
This sequence seems problematic with just one input used for all limit switches.

Is it possible to change the homing sequence to Z-X-Y or Z-Y-X?

**Comment by terjeio** [comment]
> Is it possible to change the homing sequence to Z-X-Y or Z-Y-X?

Yes, set `$45=2` and `$46=1`

**Reply by Switch-Witch** [reply]
Thank you so much - again 😀
